LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/IT87XX.cs

```diff
@@ -158,8 +159,15 @@ Chip.IT8655E or
                 Controls = new float?[6];
                 break;
 
+            case Chip.IT8696E:
+                Voltages = new float?[10];
+                Temperatures = new float?[5];
+                Fans = new float?[5];
```

Comment:
Based off the data entries in #1510, this Fans and Controls should probably have 6 instead of 5

Reply:
Thanks, changed array size from 5 to 6